modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro

```diff
@@ -69,6 +69,23 @@ definition {
 		WaitForPageLoad();
 	}
 
+	@summary = "Edit total MDF amount field"
+	macro editTotalMDFValue {
+		if (!(isSet(activitiyIndex))) {
+			var activityIndex = 0;
+		}
+
+		Type(
+			key_fieldName = "activities[0].mdfRequestAmount",
+			locator1 = "PRMMDFRequest#TEXT_FIELD",
+			value1 = ${newValue});
+
+		KeyPress(
+			key_fieldName = "activities[0].mdfRequestAmount",
+			locator1 = "PRMMDFRequest#TEXT_FIELD",
+			value1 = "\TAB");
```

Comment:
Hi @mateussandes23, the activity index should be a variable: `${activityIndex}`

Reply:
Hi @mateussandes23, update the [commit message](https://github.com/pt-liferay-solutions/liferay-portal/pull/3228/commits/e3e644438527b1805f8ce129f67e92fdce7e29a0) too.